app/src/main/java/dev/msfjarvis/aps/util/extensions/AndroidExtensions.kt

```diff
@@ -63,9 +63,6 @@ val Context.clipboard
 /** Wrapper for [getEncryptedPrefs] to avoid open-coding the file name at each call site */
 fun Context.getEncryptedGitPrefs() = getEncryptedPrefs("git_operation")
```

Comment:
@msfjarvis you may also want to remove this, since it is available using the DI framework.

Reply:
It's used in `SshKey` and I didn't want to change it entirely yet.